setup.py

```diff
@@ -51,12 +51,16 @@
         "pytest-cov==2.5.1",
         "pytest-xdist==1.18.1",
         "pytest-watch>=4.1.0,<5",
-        "termcolor==1.1.*"
```

Comment:
Was `termcolor` first added by the other benchmark PR?

Reply:
Yep, it was